packages/@react-spectrum/picker/test/Picker.test.js

```diff
@@ -1624,4 +1624,38 @@ describe('Picker', function () {
       expect(progressbar).not.toBeInTheDocument();
     });
   });
+
+  describe('disabled', function () {
+    it('enables the hidden select when isDisabled is false', function () {
```

Comment:
this isn't really a 'disabled' test, could you move this assert to the default test?